.github/workflows/integration_test_arm64.yaml

```diff
@@ -0,0 +1,17 @@
+name: Integration tests arm64
+
+on:
+  pull_request:
+
+jobs:
+  integration-tests-arm64:
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    secrets: inherit
+    with:
+      channel: 1.28-strict/stable
+      modules: '["test_cert_relation", "test_ingress_relation", "test_nginx_route"]'
+      juju-channel: 3.1/stable
+      self-hosted-runner: true
+      self-hosted-runner-arch: arm64
```

Comment:
Can you also specify the edge label (or check if its used per default in operator workflows)? We need people to test arm edge runners (and in general specifying all labels is encouraged, Otherwise an arbitrary size is used (medium, large, xlarge).